packages/fxa-auth-server/lib/senders/emails/fluent-localizer.ts

```diff
@@ -78,7 +78,7 @@ class FluentLocalizer {
     let selectedLocale: string = 'en-US';
 
     async function* generateBundles(currentLocales: Array<string>) {
-      let bundle = new FluentBundle(currentLocales);
+      let bundle = new FluentBundle(currentLocales, { useIsolating: false });
```

Comment:
I think by default, fluent inserts unicode characters while translating the variables so disabling the `useIsolating` option can help us avoid that. Reference: [fluent docs](https://projectfluent.org/fluent.js/bundle/classes/fluentbundle.html)

Reply:
What did this look like without vs with? From reading the docs it seems fine to set this to `false` since we are setting the text-direction with explicit markup which it seems to recommend, but just curious what it's outputting otherwise.

Reply:
Remember I was talking about some test failures happening locally last week? We introduced usage of `selectors` with our fluent setup so when fluent was translating those selectors, my guess is that it was also adding some unicode characters around the text. 
If you still want to see how localization happens otherwise, turn useIsolating to true which is basically the default value and run the `mjml-emails` tests. You'll find the tests failing despite the expected text matching the actual one
Also, see this [issue](https://github.com/projectfluent/fluent.js/issues/484)

Reply:
Oh I see! Thanks for the explaination. 🙂